gunicorn/arbiter.py

```diff
@@ -63,6 +63,7 @@ def __init__(self, app):
         self.reexec_pid = 0
         self.master_pid = 0
         self.master_name = "Master"
+        self.stopping = False
```

Comment:
Not sure but may be `is_stopping`  more better name for this, isn't it?

Reply:
probably yes... but I'll wait for the more fundamental problems that @benoitc raised before dealing with the details